src/main/java/org/codehaus/plexus/archiver/zip/AbstractZipArchiver.java

```diff
@@ -52,7 +52,7 @@
 {
     "NullableProblems", "UnusedDeclaration"
 } )
-public abstract class AbstractZipArchiver
+public abstract class  AbstractZipArchiver
```

Comment:
Redundant space

Reply:
nice catch: I could not see it while reviewing before committing :)